Raw json:

{'hasError': False, 'errParameter': [], 'reason': []}

{'hasError': True, 'errParameter': ['fs.ftp.impl', 'fs.s3a.connection.ssl.enabled', 'fs.s3a.multipart.purge.age', 'ipc.[port_number].weighted-cost.response', 'hadoop.http.cross-origin.max-age', 'ha.health-monitor.rpc.connect.max.retries'], 'reason': ['fs.ftp.impl is not a valid property', 'fs.s3a.connection.ssl.enabled is not a valid property', 'fs.s3a.multipart.purge.age is not a valid property', 'ipc.[port_number].weighted-cost.response is not a valid property', 'hadoop.http.cross-origin.max-age is not a valid property', 'ha.health-monitor.rpc.connect.max.retries is not a valid property']}

{'hasError': True, 'errParameter': ['hadoop.security.group.mapping.ldap.bind.users'], 'reason': ['The property hadoop.security.group.mapping.ldap.bind.users is not supported in Hadoop Common 3.3.0.']}

Raw json:

{'hasError': False, 'errParameter': [], 'reason': []}

{'hasError': True, 'errParameter': ['fs.ftp.impl'], 'reason': ['fs.ftp.impl is not a valid property for Hadoop Common version 3.3.0']}

{'hasError': False, 'errParameter': [], 'reason': []}

Final result:

The CONFIGURATION FILE IS CORRECT
